src/ripple/core/JobCounter.h

```diff
@@ -159,8 +155,7 @@ class JobCounter
         // construct the wrapped job if we were not previously joined.
         // That would introduce a timing window.
         boost::optional<CountedJob<JobHandler>> ret {
-            CountedJob<JobHandler> {
-                *this, std::forward<JobHandler> (jobHandler)}};
+            CountedJob<JobHandler> {*this, std::move (handler)}};
```

Comment:
There's an argument to be made either way, and I won't loose any sleep on it no matter which way you go.  But if I were to come across this code a few months from now, and read `forward` combined with "can never be an lvalue", I would be inclined to believe that the coder didn't understand what he was doing, and that either the `forward` was wrong, or the restriction is wrong.  If you stick with the `forward`, I recommend a comment indicating that you're aware that it will always be forwarding as an rvalue.